### DISCUSSION TITLE
Повысить урон на мак-10

### DISCUSSION THREAD

**Discussion by FSBJI** [octothorp-team/public-dobrograd#616] [Предложения]
Я считаю что нужно ему повысить урон, так как с него не возможно убивать, 11 урона в тело? Особенно когда пули раз через раз регают, предлагаю повысить с 11 до 16.

**Comment by JunkCoded** [comment]
Пример отличного рега с MAC 10 - https://www.youtube.com/watch?v=gT4408y-_0o
Когда засчитывает хит воспроизводиться звук
